docs/manual/generics.tex

```diff
@@ -853,6 +856,95 @@
   void m(@Nullable Object obj)
 \end{Verbatim}
 
+\sectionAndLabel{Classes with implicit qualifier parameters}{class-qualifier-polymorphism}
+
+The Checker Framework supports type \emph{qualifier} polymorphism for classes.  This is similar to
```

Comment:
Could you be more specific than "similar to"?

Maybe giving a use case would be the best way to introduce the feature.

Is the key point of the feature to enable a field or method access to have the same type qualifier as the receiver argument (or at least to be controlled by the type of the receiver argument)?
That is, to make the type of `x.f` depend on the type of `x`?  And the same for a method parameter or return type?

Java generics can be viewed as giving a way to control method/field types by the type of the (type argument in the) receiver.

Reply:
I'm struggling to address this comment.  Would you please use [this doc](https://docs.google.com/document/d/1nbJ4eTEs2kpysMbRyd0DJWH-BKuXawA77woncVgelGc/edit?usp=sharing) and see if you can address this?